docs/studio/groups/group-rules.mdx

```diff
@@ -0,0 +1,83 @@
+---
+title: "Group Rules"
+description: "Configure the resources accessible to your groups."
+icon: "database"
+---
+
+A **group rule** defines the roles and associated resources that determine what group members and API keys can access, as well as the level of permissions granted.
+
+<Frame>
+    <img src="/images/studio/group-rules.png" />
+</Frame>
+
+## Roles
+
+<Note>
+    If a role is assigned to a single resource and that resource is deleted, the role will default to granting access to all resources in the organization.
+</Note>
+
+You can assign multiple roles to a group using the `Add rule` button. If no group rules are configured, group members will not have access to any resources.
+
+<Frame caption="Role selector">
+    <img src="/images/studio/group-role-selector.png" />
+</Frame>
+
+Each role can be added only once per group. After assigning a role, you may associate it with multiple resources, but you cannot create additional rules for the same role.
+
+Roles in Cosmo are categorized into the following groups:
+
+### Organization Roles
+
+These roles apply at the organization level and cannot be limited to specific resources:
```

Comment:
Let's underline with a few roles why they are important and so useful. Let's give them real world examples.